src/lib/pages/frontsite/DocumentRequests/DocumentRequestForm.js

```diff
@@ -22,7 +22,8 @@ class DocumentRequestForm extends Component {
     this.state = {
       data: {
         title: getIn(props, 'location.state.queryString', ''),
-        medium: getIn(props, 'location.state.medium', 'PAPER'),
+        medium: getIn(props, 'location.state.formData.medium', 'PAPER'),
```

Comment:
the medium default value cannot be hardcoded, because it is a part of vocabulary, which means that depending on a configuration `PAPER` value might change

Reply:
what do you suggest for this?